great_expectations/init_notebooks/create_expectations_for_spark_dataframes.ipynb

```diff
@@ -1,310 +0,0 @@
-{
```

Comment:
Adding "head" to all dataset classes removed the need to have datasource-type-specific notebooks for creating expectations.